che-editors.yaml

```diff
@@ -287,10 +287,16 @@ editors:
       - name: che-machine-exec
         container:
           image: 'quay.io/eclipse/che-machine-exec:7.50.0'
+          memoryLimit: 128Mi
+          memoryRequest: 32Mi
+          cpuLimit: 500m
+          cpuRequest: 30m
           command:
             - /go/bin/che-machine-exec
             - '--url'
             - '127.0.0.1:3333'
+            - '--idle-timeout'
+            - '15m'
```

Comment:
As I remember, during today's standup call we discussed increasing it to 30 min.
no?

Reply:
It has been increased to 30 minutes from the che-operator side in this PR: https://github.com/eclipse-che/che-operator/pull/1443

as defaults here: https://github.com/eclipse-che/che-operator/blob/fdbbcdb47e811a54e8e3672ba856d43a3aa5a3e0/api/v2/checluster_types.go#L96-L97